nilearn/__init__.py

```diff
@@ -1,6 +1,6 @@
 """
 Machine Learning module for NeuroImaging in python
-==================================================
+--------------------------------------------------
```

Comment:
Sorry, I'm just surprised about that one. Is there any rule somewhere about that ?
also, python -> Python